Cubical/Data/DescendingList/DescendingList.agda

```diff
@@ -0,0 +1,285 @@
+------------------------------------------------------------------------
+-- Descending lists
+--
+-- Anders Mörtberg and Chuangjie Xu, October 2019
+--
+-- We define descending lists via simultaneous definitions and show
+-- that they are isomorphic to finite multisets.  The conversion from
+-- finite multisets to descending lists is exactly insertion sort.  We
+-- obtain the concatenation operation on descending lists and its
+-- properties by transporting those on finite multisets.
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical --safe #-}
+
+open import Cubical.Data.Empty
+open import Cubical.Data.Unit
+
+open import Cubical.Relation.Nullary
+open import Cubical.Relation.Nullary.DecidableEq
+
+open import Cubical.Foundations.Everything
+
+open import Cubical.Core.Everything
+
+open import Agda.Builtin.List using (List ; [] ; _∷_)
```

Comment:
You can import https://github.com/agda/cubical/blob/master/Cubical/Data/List/Base.agda instead

Reply:
Could you do this change please?

Reply:
I did. But I just noticed that what I did is
`open import Cubical.Data.List using (List ; [] ; _∷_)`
Or shall I change it use Base instead?

Reply:
Oh I see, I guess that's fine! Thanks